modules/programs/vicinae.nix

```
@@ -73,35 +85,47 @@ in
     };
 
     themes = lib.mkOption {
-      inherit (jsonFormat) type;
+      type = lib.types.nullOr tomlFormat.type;
```

```suggestion
      inherit (tomlFormat) type;
```
Both are essentially the same type.

---

Yeah, I getcha.
As stated in the commit description:
> Solution based off of ashell's solution

Ashell's settings option uses `nullOr tomlFormat.type` so I just kinda pulled from that